db/seeds.rb

```diff
@@ -58,3 +58,15 @@
                activated_at: Time.zone.now
   )
 end
+
+20.times do
+  Author.create!(name: Faker::Name.name,
+                 description: Faker::Lorem.sentence(word_count: 30)
+  )
+end
+
+50.times do
+  BookAuthor.create!(author_id: Faker::Number.between(from: 1, to: 20),
```

Comment:
`Faker::Number.between(from: 1, to: 20)` random này hơi phiêu, rõ ràng em có thể lấy id của author trong DB mà dùng ấy 
`Author.all.pluck(:id).sample` hoặc em lấy data từ author em tạo book author cũng dc mà
dùng association nè `author.book_authors.create book_id: ...`